src/Jackett.Common/Definitions/torrentmasters.yml

```diff
@@ -0,0 +1,151 @@
+---
+id: torrentmasters
+name: TorrentMasters
+description: "TorrentMasters is a Hungarian private tracker for MOVIES / TV / GENERAL"
+language: hu-HU
+type: private
+encoding: UTF-8
+links:
+  - https://torrentmasters.info/
+
+caps:
+  categorymappings:
+    - {id: 92, cat: Books, desc: "e-Book (ENG)"}
+    - {id: 78, cat: Books, desc: "e-Book (HUN)"}
+    - {id: 80, cat: Other, desc: "Egyéb"}
+    - {id: 85, cat: Movies/DVD, desc: "Film - DVD/9 (ENG)"}
+    - {id: 66, cat: Movies/DVD, desc: "Film - DVD/9 (HUN)"}
+    - {id: 84, cat: Movies/HD, desc: "Film - HD (ENG)"}
+    - {id: 68, cat: Movies/HD, desc: "Film - HD (HUN)"}
+    - {id: 82, cat: Movies, desc: "Film - XviD (ENG)"}
+    - {id: 64, cat: Movies, desc: "Film - XviD (HUN)"}
+    - {id: 72, cat: PC/Games, desc: "Játék - ISO"}
+    - {id: 73, cat: Console, desc: "Játék - Konzol"}
+    - {id: 94, cat: PC, desc: "Játék - RIP"}
+    - {id: 79, cat: PC/Mobile-Other, desc: "Mobil/PDA"}
+    - {id: 95, cat: PC, desc: "Programok - Egyéb (Win)"}
+    - {id: 74, cat: PC/ISO, desc: "Programok - ISO (Win)"}
+    - {id: 75, cat: PC/Mac, desc: "Programok - Linux/Unix/Mac OS"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (ENG)"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (HUN)"}
```

Comment:
The id for dvd/hun  appears to be a duplicate of the dvd/eng (or visa-versa). Please correct.

Reply:
Fixed, my bad.